projects/plugins/jetpack/modules/contact-form.php

```diff
@@ -18,6 +20,11 @@
  * Additional Search Queries: contact, form, grunion, feedback, submission, contact form, email, feedback, contact form plugin, custom form, custom form plugin, form builder, forms, form maker, survey, contact by jetpack, contact us, forms free
  */
 
+if ( defined( 'JETPACK_FORMS_PACKAGE_ENABLED' ) && JETPACK_FORMS_PACKAGE_ENABLED ) {
```

Comment:
Could we have this be a filter instead? something like

```diff
/**
 * Whether to load the newer Jetpack Forms package.
 *
 * @use add_filter( 'jetpack_contact_form_use_package', '__return_true' );
 * @module contact-form
 *
 * @since $$next-version$$
 *
 * @param bool $load_contact_form_package Load Jetpack Forms package. Default to false.
 */
if ( apply_filters( 'jetpack_contact_form_use_package', false ) ) {
```

Reply:
Hmm, I guess we could. Just one general question, where would we best define this later when we do want it to be true?

Reply:
Folks usually rely on a functionality plugin to keep those in their development environment. I personally have a `wp-content/plugins/helper/helper.php` plugin on my local dev environment for example.

We can then leverage that same filter in production environments. One could imagine a simple mu-plugin on WordPress.com Simple that would do something like the following during a call for testing, for example:

```php
add_filter(
    'jetpack_contact_form_use_package',
    function () {
        if ( is_automattician() ) {
			return true;
        }
        return false;
    }
);
```

On WoA sites, this would be added to Automattic/wpcomsh instead.

And for folks testing on WoA sites or on testing sites later, that may not have file access to modify `wp-config.php` or may not know how to do it, they can use a plugin like [code snippets](https://jetpack.com/support/adding-code-snippets/) to add the snippet when testing.